src/outbackcdx/Webapp.java

```diff
@@ -194,9 +204,56 @@ private String formatStackTrace(Exception e) {
         StringWriter stacktrace = new StringWriter();
         e.printStackTrace(new PrintWriter(stacktrace));
         return stacktrace.toString();
+    }
+    
+    Response changeFeed(Web.Request request) throws Web.ResponseException, IOException {
+        String collection = request.param("collection");
+        long since = Long.parseLong(request.param("since"));
+        if(verbose) {
+            out.println(String.format("collection: %s, since: %s", collection, since));
+        }
+        final Index index = getIndex(request);
+        out.println(String.format("got index for collection %s since %s", collection, since));
+
+        TransactionLogIterator logReader = index.getUpdatesSince(since);
```

Comment:
`TransactionLogIterator` also looks like it should be closed.

```java
try (TransactionLogIterator logReader = index.getUpdatesSince(since)) {
    ...
}
```